src/styles/annotator/adder.scss

```diff
@@ -102,21 +102,19 @@ $adder-transition-duration: 80ms;
   @include focus.outline-on-keyboard-focus;
 
   box-shadow: none;
-  font-family: h;
```

Comment:
I imagine this was related to the icon font.

Reply:
Yes, I believe so.